cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -374,7 +374,7 @@ public String parseLogging(Logging logging, ConfigMap externalCm) {
      * @return The generated ConfigMap.
      */
     public ConfigMap generateMetricsAndLogConfigMap(ConfigMap cm) {
-        Map<String, String> data = new HashMap<>();
+        Map<String, String> data = new HashMap<>(3);
```

Comment:
Might be missing something here, but don't we only add two things to `data` , not three?